R/add_ci.R

```diff
@@ -0,0 +1,283 @@
+#' Add Proportion CIs
+#'
+#' Add a new column with the confidence intervals for proportions.
+#'
+#' @param x A `tbl_summary` object
+#' @param statistic Formula indicating how the confidence interval will be placed.
+#' Default is `list(all_categorical() ~ "{conf.low}%, {conf.high}%", all_continuous() ~ "{conf.low}, {conf.high}")`
+#' @param method Confidence interval method. Default is
+#' `list(all_categorical() ~ "wilson", all_continuous() ~ "t.test")`.
+#' Must be one of
+#' `c("wilson", "wilson.no.correct", "exact", "asymptotic")` for categorical
+#' variables, and `"t.test"` for continuous variables.
+#' See details below.
+#' @param conf.level Confidence level. Default is `0.95`
+#' @param ci_fun Function to style upper and lower bound of confidence
```

Comment:
Could `ci_fun` be confusing, i.e. corresponding to the function to compute a CI and not a function to format/style the values?

To be consistent with other gtsummary functions, calling this parameter `style_fun`?

Reply:
updated to style_fun!~